include/sancus_support/sancus_step.h

```diff
@@ -78,7 +80,64 @@ void timerA_isr_entry(void)                                         \
             "m"(__ss_dbg_measuring_reti_latency),                   \
             "i"(TACTL_CONTINUOUS),                                  \
             "i"(TACTL_DISABLE),                                     \
-            "i"(TACTL_ENABLE)                                       \
+            "i"(TACTL_ENABLE),                                      \
+            "i"(TACCTL_ENABLE_CONT),                                \
+            "m"(TACCTL0),                                           \
+            "i"(TACTL_ENABLE_CONT)                                  \
+            :                                                       \
+            );                                                      \
+}
+
+#define SANCUS_STEP_ISR_ENTRY2(fct_single_step, fct_end)            \
```

Comment:
hmm, this seems to be verbatim duplicated from SANCUS_STEP_ISR_ENTRY (?) Is it possible to create a common top-level macro with an extra `irq_nb` argument to avoid unnecessary code duplication? :-)